src/rpc/server.cpp

```diff
@@ -255,35 +256,29 @@ static const CRPCCommand vRPCCommands[] = {
 
 CRPCTable::CRPCTable()
 {
-    unsigned int vcidx;
-    for (vcidx = 0; vcidx < (sizeof(vRPCCommands) / sizeof(vRPCCommands[0])); vcidx++)
-    {
-        const CRPCCommand *pcmd;
-
-        pcmd = &vRPCCommands[vcidx];
-        mapCommands[pcmd->name] = pcmd;
-    }
+    for (auto cmd : vRPCCommands)
+        appendCommand(cmd);
 }
 
 const CRPCCommand *CRPCTable::operator[](const std::string &name) const
 {
-    map<string, const CRPCCommand *>::const_iterator it = mapCommands.find(name);
+    map<string, CRPCCommand>::const_iterator it = mapCommands.find(name);
     if (it == mapCommands.end())
         return NULL;
```

Comment:
while at it why not using `nullptr` here?

Reply:
Good point. Changed. I also replaced another `pair<...>` with the `rpc_pair` typedef.